### PR TITLE
[AF-30] Add apps in organization sidebar 

### PR DESCRIPTION
**Pull request by sadia**
:koala: 

Add organization_sidebar to locations

Relates to:
* Console - zendesk/zendesk_console#8401
* ZAF - zendesk/zendesk_app_framework#425

org sidebar

/cc @zendesk/quokka

### References
 - Jira link: https://zendesk.atlassian.net/browse/AF-30

### Risks
 - None

### CODE REVIEW

**Comment by sandlerr** [comment]
:+1:

**Comment by svizzari** [comment]
:+1:

File: lib/zendesk_apps_support/validations/manifest.rb [hunk 1]
```diff
@@ -5,7 +5,7 @@ module Validations
     module Manifest
       REQUIRED_MANIFEST_FIELDS = %w( author defaultLocale ).freeze
       OAUTH_REQUIRED_FIELDS    = %w( client_id client_secret authorize_uri access_token_uri ).freeze
-      LOCATIONS_AVAILABLE      = %w( top_bar nav_bar ticket_sidebar new_ticket_sidebar user_sidebar ).freeze
+      LOCATIONS_AVAILABLE      = %w( top_bar nav_bar ticket_sidebar new_ticket_sidebar user_sidebar organization_sidebar).freeze
```
Comment (princemaple) [inline lib/zendesk_apps_support/validations/manifest.rb]: missing a space

Comment (adammw) [inline lib/zendesk_apps_support/validations/manifest.rb]: ^^ +1

**Comment by danielbreves** [comment]
:+1:

**Comment by adammw** [comment]
:+1:

**Pull request merged**